src/kde_material_you_colors/main.py

```diff
@@ -479,9 +502,12 @@ def main():
                 logging.error(f"Could not get wallpaper {str(wallpaper.error)}")
             if plugin_watcher.changed and wallpaper.type == "screenshot":
                 head = "Screenshot mode enabled"
+            if config.read("manual_fetch"):
+                cont = "Manual color fetch enabled, press 'Fetch colors' in the widget to take a new screenshot"
+            else:
                 cont = f"Waiting {config.read('screenshot_delay')}s between updates"
-                notify.send_notification(head, cont)
-                logging.warning("%s, %s", head, cont)
+            notify.send_notification(head, cont)
+            logging.warning("%s, %s", head, cont)
```

Comment:
This causes the back-end to terminate with `UnboundLocalError: cannot access local variable 'head' where it is not associated with a value` error if `if plugin_watcher.changed and wallpaper.type == "screenshot":` is not true, I would change it similar to what you did on line 404

```py
if plugin_watcher.changed and wallpaper.type == "screenshot":
    head = "Screenshot mode enabled"
    if config.read("manual_fetch"):
        cont = "Manual color fetch enabled, press 'Fetch colors' in the widget to take a new screenshot"
    else:
        cont = f"Waiting {config.read('screenshot_delay')}s between updates"
    notify.send_notification(head, cont)
    logging.warning("%s, %s", head, cont)
```